### DISCUSSION TITLE
Refer to values in ConfigMap and Secret resources from Application

### DISCUSSION THREAD

**Discussion by heyrun** [argoproj/argo-cd#14850] [Q&A]
I want to deploy a remote helm chart to my cluster and within the helm values, I need to specify specific tokens that I have already created as secrets ( managed by the external secret operator). 

How can I reference the values from the Secrets or ConfigMap in the `helm` or `kustomize` section of ArgoCD Application? 

For reference, with FluxCD I can use [valuesFrom](https://fluxcd.io/flux/guides/helmreleases/#refer-to-values-in-configmap-and-secret-resources) to reference a value in ConfigMap or Secret. How can I achieve the same with the Application?

**Comment by chauncey-garrett** [comment]
@heyrun this may help: 
https://github.com/argoproj/argo-cd/discussions/11353#discussioncomment-4208462

I know the know the pattern works for SSO which can be configured with ArgoCD's `ConfigMap` so I'd hope that the same pattern could be applied elsewhere. I haven't tested though so please post back what you find if you try!
